src/vs/workbench/contrib/chat/browser/chatSessions/view/sessionsTreeRenderer.ts

```diff
@@ -336,6 +337,16 @@ export class SessionsRenderer extends Disposable implements ITreeRenderer<IChatS
 			templateData.timestamp.ariaLabel = session.relativeTimeFullWord ?? '';
 			templateData.timestamp.parentElement!.classList.toggle('timestamp-duplicate', session.hideRelativeTime === true);
 			templateData.timestamp.parentElement!.style.display = '';
+
+			// Add tooltip showing full date/time when hovering over the timestamp
+			if (session.timing?.startTime) {
+				const fullDateTime = getLocalHistoryDateFormatter().format(session.timing.startTime);
+				templateData.elementDisposable.add(
+					this.hoverService.setupDelayedHover(templateData.timestamp, {
+						content: `${nls.localize('chat.sessions.lastActivity', 'Last Activity: {0}', fullDateTime)}`
```

Comment:
I don't think this needs to be wrapped in a string. We can use the call to `localize` directly